pom.xml

```diff
@@ -13,6 +13,7 @@
         <endorsed.dir>${project.build.directory}/endorsed</endorsed.dir>
 	<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 	<additionalparam>-Xdoclint:none</additionalparam>
+        <compilerArgument></compilerArgument>
```

Comment:
What is this empty "compilerArgument" for?

Reply:
Please see the commit message of 8d5a25fa0913.

>  IDEA 2018.2 fails to build the project because of the uninitialized Maven property 'compilerArgument'. The property name was passed as argument string to the compiler, which of course failed. Adding an empty default fixes this.

Reply:
Very strange. Can you please add a comment about this?

Reply:
Will do so. :smile: